config/settings/common.py

```diff
@@ -236,6 +238,8 @@
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_EMAIL_VERIFICATION = 'mandatory'
 
+ACCOUNT_DEFAULT_HTTP_PROTOCOL = env.str("DJANGO_ACCOUNT_DEFAULT_HTTP_PROTOCOL", default="http")
```

Comment:
Hmm, true. I'll probably move this bit to production and set the default there to be `https`. `http` is the default for this setting in `django-allauth` so left it like that here since that might be nice for testing purposes.